modules/dynamic-plug-in-sdk-extensions.adoc

```diff
@@ -1612,8 +1591,7 @@ graph element.
 |Name |Value Type |Optional |Description
 |`link`
 |`CodeRef<(element: GraphElement) => React.Component | undefined>` |no
-|Return the resource link if provided, otherwise undefined.Use
-ResourceIcon and ResourceLink for styles.
+|Return the resource link if provided, otherwise undefined. Use `ResourceIcon` and `ResourceLink` for styles.
```

Comment:
Something like this? 
```suggestion
|Return the resource link if provided, otherwise undefined. Use the `ResourceIcon` and `ResourceLink` properties for styles.
```